models/xero__profit_and_loss_report.sql

```diff
@@ -18,12 +18,13 @@ with calendar as (
         ledger.account_code,
         ledger.account_type, 
         ledger.account_class, 
+        ledger.source_relation, 
```

Comment:
Did not think about that 🤦‍♀️ I updated the unique test in the yml back to the `profit_and_loss_id` as well since source relation will have already been included in there